src/core_plugins/kibana/public/dashboard/index.js

```diff
@@ -114,6 +114,13 @@ uiRoutes
             return savedDashboard;
           })
           .catch((error) => {
+            // A corrupt dashboard was detected (e.g. with invalid JSON properties)
+            if (error instanceof InvalidJSONProperty) {
+              new Notifier().warning(error);
```

Comment:
How about using `toastNofications` instead of Notifier?  We are trying to remove Notifier. 

Here is an example of where toastNotifications are used to display a dashboard load error https://github.com/elastic/kibana/blob/master/src/core_plugins/kibana/public/dashboard/dashboard_app.js#L306

Reply:
I'm down with that. That's actually what I originally did, then I saw that notifier gave a much more prominent error, which I thought was helpful. But if we're ditching notifier, will do.